coalib/bears/Bear.py

```diff
@@ -99,7 +99,9 @@ class Bear(Printer, LogPrinter):
     and the CAN_DETECT sets. Possible values:
 
     >>> CAN_DETECT = {'Syntax', 'Formatting', 'Security', 'Complexity', 'Smell',
-    ... 'Redundancy', 'Simplification', 'Variable Misuse', 'Spelling', 'Other'}
+    ... 'Unused Code', 'Simplification', 'Variable Misuse', 'Spelling',
+    ... 'Memory Leak', 'Documentation', 'Duplication', 'Commented Code',
```

Comment:
none use Variable Misuse either, but its something a bear should probably do, so keep it :D ?